src/utils/fees.ts

```diff
@@ -46,46 +60,132 @@ function isEIP1559Fees(chain: ChainId) {
   return chain === ChainId.Ethereum || chain === ChainId.Polygon || chain === ChainId.Avalanche;
 }
 
+/*
+ * Probable fee for EIP1559 (BASE + TIP)
+ */
+function probableFeePerUnitEIP1559(suggestedGasFee: EIP1559Fee) {
+  return suggestedGasFee.suggestedBaseFeePerGas || 0 + suggestedGasFee.maxPriorityFeePerGas;
```

Comment:
`0 + ` is a quirky way of I presume converting a string to a number? Can we make it more explicit such as `Number.parseFloat()`

Reply:
logical operators have greater priority than arithmetic operators, not sure if the prettier won't get rid of ( ) if you use them for better readability.

Reply:
> In case suggestedBaseFeePerGas is undefined (which highly likely), then it is replaced by 0.
Ok that's even more unpredictable than I expected then 😅 
Is there a way to write this so I can understand you are expecting 0 when it's undefined

Reply:
Done

Reply:
Done